elasticassignment/ageofempires/management/commands/command_classes.py

```diff
@@ -0,0 +1,206 @@
+from abc import abstractmethod
+from elasticsearch.helpers import bulk
+import requests
+import elasticsearch
+from django.conf import settings
+
+class AOElasticsearch:
+
+    def make_connection():
+        return elasticsearch.Elasticsearch([settings.CONFIG])
+
+    def getAPIdata(url:str):
+        response_data = requests.get(url)
+        return response_data.json()
+
+    @abstractmethod
+    def make_index():
+        pass
+
+    @abstractmethod
+    def populate_index(url:str):
+        pass
+
+
+class Civilizations(AOElasticsearch):
```

Comment:
please create a package `models` and put all model classes inside in their own files.